examples/language-analyzer-v0.5.0.py

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env python3
```

Comment:
imo we don't need to include this example. But add a test that checks "normal" grpc code works.

Reply:
ok, gonna remove it. BTW how do you propose to test it? I mean the normal gRPC datastub should be guaranteed to work by gRPC itself, isn't it?

Reply:
it doesn't work right now on master. So we can break it again in the future occasionally.

I propose something like:

```python
class DummyAnalyzer(pb.AnalyzerServicer):
    def NotifyReviewEvent(self, request, context):
        return pb.EventResponse(messages=["review"])

    def NotifyPushEvent(self, request, context):
        return pb.EventResponse(messages=["push"])


class Test:
   def test_review():
      res = stub.NotifyReviewEvent(event_pb2.ReviewEvent())
      self.assertEqual(res.messages, ["review"])
```